index.js

```diff
@@ -7,17 +7,15 @@ class ShoppingCart {
     this.total += price * quantity;
     if (this.items[itemName] !== undefined) {
```

Comment:
Change this line to `if (!this.items.itemName)` (No need for the `!==undefined`)

Reply:
you sure it is not suppose to be this.items.itemName? putting the ! means it doesnt exist. And am checking if it does

Reply:
Oh, you're right.